README.md

```diff
@@ -36,6 +36,14 @@ https://github.com/decidim/decidim/blob/master/docs/getting_started.md
 Then install PostGIS:
 https://postgis.net/install/
 
+### Setup (Docker)
```

Comment:
We need to figure out if Docker should be the One True Way™ to set this up. I think it should be—if you agree, could you remove the other setup instructions and associated files? If not, the Docker instructions should still go first as our "supported" install method.